examples/meta/src/kernel/inverse_multiquadric.sg.in

```diff
@@ -0,0 +1,19 @@
+File traindat = read_csv("@SHOGUN_DATA@/fm_train_real.dat")
```

Comment:
multi_quadratic

Reply:
Do you mean change file name to multi_quadratic?

Reply:
Yes both example and data

Reply:
sorry, I see now. I of coure mean "inverse_multi_quadratic"
I should have been more clear, sorry about that hassle :)